linux/web_tools.bsh

```diff
@@ -41,7 +41,17 @@ function download_to_stdout()
   elif command -v curl &> /dev/null; then
     curl -fsSL "${1}"
   elif command -v python &> /dev/null; then
-    python -c $'try:\n import urllib2 as u\nexcept:\n import urllib.request as u\nimport os; os.write(1,u.urlopen("'"${1}\").read())"
+    python -c  'if 1:
```

Comment:
if True?

Reply:
The one of your tricks, so that the multi line string can be indented in bash to look pretty